addons/hw_drivers/iot_handlers/drivers/PrinterDriver_L.py

```diff
@@ -366,60 +367,74 @@ def _get_iot_status(self):
 
         return { "mac": mac, "pairing_code": pairing_code, "ssid": ssid, "ips": ips }
 
-    def print_status(self):
-        if not self.connected_by_usb:
+    def print_status(self, data=None):
+        """Prints the status ticket of the IoT Box on the current printer.
+
+        :param data: If not None, it means that it has been called from the action route, meaning
+        that no matter the connection type, the printer should print the status ticket.
+        """
+        if not self.connected_by_usb and not data:
             return
         if self.device_subtype == "receipt_printer":
             self.print_status_receipt()
         if self.device_subtype == "label_printer":
             self.print_status_zpl()
```

Comment:
Maybe a switch case with some function like this ?
            `self.print_status_raw_status()` (put only char)

Reply:
By this way you can use office_printer to print status